src/services/userService.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const apiUrl = 'http://localhost:8080/v1';
+
+export async function searchUser(userInput) {
+  return axios.post(`${apiUrl}/searchEmails`, { userInput });
```

Comment:
Have you tried this and it works? I wonder where Axios sets/gets the auth token.

Reply:
it actually works, I think the auth token is handled by the axios interceptor @line 20 of eventService.js.

Reply:
Does it mean userService now is depending on the envetService to call first?  Is there any way we can decouple them, (i.e. Set auth token if not yet exsited)?